### DISCUSSION TITLE
Kazanmak

### DISCUSSION THREAD

**Discussion by muhmmedcelik** [IBAX-io/go-ibax#1372] [Bug Submission]
Burada nasıl puan kazancagimiz güzelce ve açıkca anlatilsa her yerde nasıl kazanacağımiza dair araştırma yapmak zorunda kalmazdik .

**Comment by Odysseusksk** [comment]
Puan kazanmak bir araç. Amaç ise IBXC kazanmak. Bunun 3 yolu var. İlki RT ederek puan kazanarak, kazanılan puanlar sonucu sıralamaya girerek coin kazanmak. İkincisi, Referans yöntemi ile direkt coin kazanmak ve son olarak ta yine puan kazanabilmek için GitHub ta aktif olmak. Umarım açıklayıcı olabilmişimdir.
